test/upgrading/upgrader.spec.ts

```diff
@@ -619,10 +630,10 @@ describe('libp2p.upgrader', () => {
     const remoteLibp2pUpgraderOnStreamSpy = sinon.spy(remoteLibp2p.components.upgrader as DefaultUpgrader, '_onStream')
 
     const stream = await localConnection.newStream(['/echo/1.0.0'])
-    expect(stream).to.include.keys(['id', 'close', 'reset', 'stat'])
+    expect(stream).to.include.keys(['abortController', 'source', 'id', 'name'])
```

Comment:
These keys are now included in the Yamux stream, I am not sure what this test intended by specifying specific keys.